packages/svelte2tsx/src/svelte2tsx.ts

```diff
@@ -614,11 +614,7 @@ function processInstanceScriptContent(str: MagicString, script: Node): InstanceS
         ts.forEachChild(list, (node) => {
             if (ts.isVariableDeclaration(node)) {
                 if (ts.isIdentifier(node.name)) {
-                    if (node.type) {
```

Comment:
@halfnelson do you remember why you made this distinction? I removed it to enable optional props, but I fear I may have overlooked something else.